packages/contentful--app-scripts/lib/utils.js

```diff
@@ -125,8 +125,68 @@ function getActionsManifest() {
   } catch {
     console.log(
       `${chalk.red('Error:')} Invalid JSON in manifest file at ${chalk.bold(
-        DEFAULT_MANIFEST_PATH
-      )}.`
+        DEFAULT_MANIFEST_PATH,
+      )}.`,
+    );
+    // eslint-disable-next-line no-process-exit
+    process.exit(1);
+  }
+}
+
+function getDeliveryFunctionsManifest() {
+  const isManifestExists = fs.existsSync(DEFAULT_MANIFEST_PATH);
+
+  if (!isManifestExists) {
+    return;
+  }
+
+  try {
+    const manifest = JSON.parse(fs.readFileSync(DEFAULT_MANIFEST_PATH, { encoding: 'utf8' }));
+
+    if (!Array.isArray(manifest.deliveryFunctions) || manifest.deliveryFunctions.length === 0) {
+      return;
+    }
+
+    console.log('');
+    console.log(`  ----------------------------
+  Delivery functions found in ${chalk.bold(DEFAULT_MANIFEST_PATH)}.
+  ----------------------------`);
+    console.log('');
```

Comment:
we should really put this formatting in a logging function, huh?

Reply:
Yep - I remember we have something like this already somewhere